modules/programs/pubs.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.pubs;
+
+in {
+  options.programs.pubs = {
+    enable = mkEnableOption "pubs";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.pubs;
+      defaultText = literalExpression "pkgs.pubs";
+      description = "The package to use for the pubs script.";
+    };
+
+    extraConfig = mkOption {
```

Comment:
This is INI format, right? We should use `pkgs.formats.ini` if so (see the swaynag module for an example).

Reply:
Not exactly, this is ConfigObj syntax which is an extension of the INI syntax with features such as `[[nested subsection]]` which are not supported by `pkgs.formats.ini`.